spec/controllers/case_logs_controller_spec.rb

```diff
@@ -43,4 +43,41 @@
       end
     end
   end
+
+  describe "submit_form" do
+    let!(:case_log) { FactoryBot.create(:case_log) }
+    let(:id) { case_log.id }
+    let(:case_log_to_submit) do
+      { "accessibility_requirements" =>
+                             %w[ accessibility_requirements_fully_wheelchair_accessible_housing
+                                 accessibility_requirements_wheelchair_access_to_essential_rooms
+                                 accessibility_requirements_level_access_housing],
+        "previous_page" => "accessibility_requirements" }
+    end
+
+    it "sets checked items to true" do
+      get :submit_form, params: { id: id, case_log: case_log_to_submit }
+      CaseLog.find(id)
+
+      expect(CaseLog.find(id)["accessibility_requirements_fully_wheelchair_accessible_housing"]).to eq(true)
+      expect(CaseLog.find(id)["accessibility_requirements_wheelchair_access_to_essential_rooms"]).to eq(true)
+      expect(CaseLog.find(id)["accessibility_requirements_level_access_housing"]).to eq(true)
```

Comment:
I think you can just do `case_log.reload` after you get request and then you can remove the 3 look ups here? Should be faster.

Reply:
Oh, I mean, would that be any different from extracting CaseLog.find(id) into a variable?

Reply:
Not really no, pretty much the same thing. Just makes it a bit more explicit it's definitely the same object I guess rather than needing to check id is the same.